starter_code/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>CSS Practice</title>
+  </head>
+  <body>
+    <header>Relaxr</header>
```

Comment:
The `<header>` tag is more of a container tag. A good use case might be something like this:

```html
<header>
  <h1>Relaxr</h1>

  <h2>Get piece of mind with a single tap</h2>

  <a href="#">Get it Now</a>
</header>
```